src/features/favorites/screen/index.tsx

```diff
@@ -1,16 +1,46 @@
 import React from 'react';
-import { SafeAreaView, Text } from 'react-native';
+import { FlatList, SafeAreaView, Text, View } from 'react-native';
 
 import { translate } from 'localization/hooks';
 
+import { useGetFavorites, useRemoveFavorite } from 'network/queries/favorite-queries';
+
+import FavoriteItem from './favoriteItem';
 import styles from './styles';
 
 const FavoritesScreen = () => {
+  const { data, refetch } = useGetFavorites();
+
+  const { mutate: removeFavorite } = useRemoveFavorite({
+    onError: error => {
+      console.log('RemoveFavorite: ', error.cause);
+    },
+    onSuccess: () => {
+      refetch();
```

Comment:
You can refetch any query by invalidating them: https://tanstack.com/query/v4/docs/framework/react/guides/query-invalidation
```
queryClient.invalidateQueries({
  queryKey: ['get-favorites'],
})
```